go.mod

```diff
@@ -87,3 +87,5 @@ require (
 	sourcegraph.com/sourcegraph/appdash v0.0.0-20180531100431-4c381bd170b4
 	sourcegraph.com/sourcegraph/appdash-data v0.0.0-20151005221446-73f23eafcf67
 )
+
+replace gihtub.com/pingcap/parser => github.com/winoros/parser v0.0.0-20190305081723-e7cb6045c53f
```

Comment:
is this change related?

Reply:
Yes, this fix needs changes from parser.